rpc-server.py

```diff
@@ -62,6 +88,7 @@ def get_random_open_port():
 
 
 if __name__ == '__main__':
+    app.secret_key = bcrypt.gensalt().decode('utf8')
```

Comment:
Where do we use this `app.secret_key`?
Any reason te rely on `bcrypt` instead of using `os.urandom`?